sections/2-project_proposals.tex

```diff
@@ -20,4 +20,4 @@ \section{Project Organization \& Enhancement Proposals}
 \end{itemize}
 
 As of the time of writing, there are a total of 8 SEPs.
```

Comment:
Pretty sure we just merged Number 9 :wink:

Reply:
good catch.